app/src/main/java/com/randomappsinc/studentpicker/home/HomeActivity.java

```diff
@@ -133,6 +139,17 @@ private void showPleaseRateDialog() {
                 .show();
     }
 
+    private void showBackupAndRestoreDialog() {
+        DialogUtil.INSTANCE.showBackupAndRestoreUpsell(this, () -> {
```

Comment:
Just curious: What does INSTANCE do?

Reply:
As DilaogUtil is an object file and we access it from Java so it deals with it as a singleton class that has one instance, once we convert HomeActivity to Kotlin we will not use INSTANCE.

The reason I used object for util is that there is no static method in Kotlin so, I made an object file that has one instance.

Reply:
Interesting, got it.